.github/workflows/release-5.x.yml

```diff
@@ -25,6 +25,33 @@ jobs:
       - if: ${{ steps.release.outputs.release_created }}
         name: actions/setup
         uses: ./.github/actions/setup
+      - if: steps.release.outputs.release_created
+        run: npm pack
+        shell: bash
+      - if: steps.release.outputs.release_created
+        name: Get release version and release package file name
+        run: |
+          PACKAGE_VERSION=$(jq '.version' package.json | tr -d '"')
+          echo "PACKAGE_VERSION=${PACKAGE_VERSION}" >> "$GITHUB_ENV"
+          echo "PACKAGE_FILE=bson-${PACKAGE_VERSION}.tgz" >> "$GITHUB_ENV"
+      - if: steps.release.outputs.release_created
+        name: Create detached signature
+        uses: mongodb-labs/drivers-github-tools/garasign/gpg-sign@main
+        with:
+          filenames: ${{ env.PACKAGE_FILE }}
+          garasign_username: ${{ secrets.GRS_CONFIG_USER1_USERNAME }}
+          garasign_password: ${{ secrets.GRS_CONFIG_USER1_PASSWORD }}
+          artifactory_username: ${{ secrets.ARTIFACTORY_USER }}
+          artifactory_password: ${{ secrets.ARTIFACTORY_PASSWORD }}
+      - if: steps.release.outputs.release_created
+        name: "Upload artifacts"
+        uses: actions/upload-artifact@v4
+        with:
+          name: ${{ env.PACKAGE_FILE }}
+          path: |
+            ${{ env.PACKAGE_FILE }}
+            ${{ env.PACKAGE_FILE }}.sig
```

Comment:
We'll need to upload the signature to the GH and make it publicly available, and it should stay there indefinitely (it will, so long as the GH release exists).  

I think we should be able to use `gh release upload` (the gh cli comes pre-installed on GHA runners): https://cli.github.com/manual/gh_release_upload

Reply:
Yep! My bad on forgetting this step